src/apiCalls.tsx

```diff
@@ -13,7 +12,6 @@ const getArt = async (objectID: number) => {
 
 const getIDs = async () => {
   const res = await fetch('https://collectionapi.metmuseum.org/public/collection/v1/search?medium=Paintings&q=painting'); 
-
   if (res.status >= 500 && res.status < 600) {
```

Comment:
This looks good at a glance to me.   I believe you're only checking for the 500 level errors because you have the known 400 level errors that you've already written a code work-around for.  This was a clever approach to a challenge all around and will be some great decision making to speak to in interviews!